demo/admin/src/App.tsx

```diff
@@ -142,6 +142,7 @@ class App extends React.Component {
                                                                                         path={`${match.path}/project-snips/main-menu`}
                                                                                         component={MainMenu}
                                                                                     />
+                                                                                    <RouteWithErrorBoundary path={`${match.path}/subsubpage`} />
```

Comment:
This should be a Dev Story in Storybook instead. Adding a useless route to Demo isn't right

Reply:
Done